go.mod

```diff
@@ -267,6 +267,7 @@ require (
 	github.com/minio/minio v0.0.0-20240312195911-24b4f9d748c4
 	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
 	github.com/relvacode/iso8601 v1.4.0
+	github.com/wkalt/migrate v0.0.0-20231211003947-1dd022ec10a3
```

Comment:
I think you didn't end up using this. If you run "go mod tidy" this change will probably disappear.